### ISSUE TITLE
verbatim ignored in interactive mode in 0.8.0

### ISSUE THREAD

**Issue by yesdave** [foundweekends/giter8#299]
When providing a `verbatim` field in default.properties and running g8 in interactive mode, g8 prompts for the verbatim field and also ignores whatever is provided, or set in the default.

In this example, `$(".snippet"` is inside a .js file.
```
$ g8 file://my-template.g8/ --force
verbatim [*.ttf *.png *.ico *.gif *.html *.js *.iml .idea]:
name [proto-app]: my-app
version [1.0.1-SNAPSHOT]:
(".snippet", []: 
```

If a default param is provided when invoking g8, the default verbatim is honoured, not asked for and works correctly by ignoring those files.

This works correctly in interactive mode in 0.7.2.

**Comment by lachatak** [comment]
I suppose it is related to the https://github.com/foundweekends/giter8/issues/288
I believe (".snippet" is not a real template variable. There is a bit calamity around $ handling.
I might be wrong. :)

**Comment by eed3si9n** [comment]
This is now "fixed" in 0.9.x branch by rolling back 0.8.0 changes.
